packages/developer-portal/src/sagas/current-member.ts

```diff
@@ -0,0 +1,75 @@
+import { put, fork, all, call, takeLatest, select } from '@redux-saga/core/effects'
+
+import { Action } from '@/types/core'
+import ActionTypes from '@/constants/action-types'
+import messages from '@/constants/messages'
+import { fetchCurrentMemberSuccess } from '@/actions/current-member'
+import { UpdateMemberModel } from '@reapit/foundations-ts-definitions'
+import { selectCurrentMemberData } from '@/selector/current-member'
+import { notification } from '@reapit/elements'
+import { fetchOrganisationMembers, updateOrganisationMemberById } from '@/services/developers'
+import { getDeveloperId, getLoggedUserEmail } from '@/utils/session'
+
+export const currentMemberFetch = function*() {
+  try {
+    const [developerId, currentMemberEmail] = yield all([call(getDeveloperId), call(getLoggedUserEmail)])
+    if (!developerId || !currentMemberEmail) {
+      throw new Error('Can not get developer ID or member email')
+    }
+    const response = yield call(fetchOrganisationMembers, { id: developerId, email: currentMemberEmail })
+    if (response && Array.isArray(response.data)) {
+      const [currentMember] = response.data
+      yield put(fetchCurrentMemberSuccess(currentMember))
+      return
+    }
+    throw new Error('Cannot fetch current member')
```

Comment:
Should show notification instead of throw error

Reply:
Same as above